### PR TITLE
Contain MultipartFile to only the REST endpoints

### PR DESCRIPTION
**Pull request by emmberk**
This PR updates both the ingest app and mis app to only use MultipartFile in the `@RestController` classes to make the apps less coupled to spring web.

### CODE REVIEW

**Comment by emmberk** [comment]
I heroed this.

**Comment by emmberk** [comment]
rebased

build now

File: ingest/src/main/java/com/connexta/ingest/client/StoreClient.java [hunk 1]
```diff
@@ -43,13 +44,26 @@ public StoreClient(
...
+      @NotNull final InputStream inputStream,
       @NotEmpty final String fileName)
       throws StoreException {
     final MultipartBodyBuilder builder = new MultipartBodyBuilder();
     builder.part("fileSize", fileSize);
     builder.part("mimeType", mimeType);
-    builder.part("file", file.getResource());
+    builder.part(
+        "file",
+        new InputStreamResource(inputStream) {
```
Comment (jhunzik) [inline ingest/src/main/java/com/connexta/ingest/client/StoreClient.java]: I wish there was a nicer way to do this

Comment (emmberk) [reply ingest/src/main/java/com/connexta/ingest/client/StoreClient.java]: I agree. I didn't research it much, so maybe I missed something.

**Pull request merged**